pom.xml

```diff
@@ -30,6 +30,8 @@
 
     <properties>
         <jenkins.version>2.401.3</jenkins.version>
+        <spotbugs.effort>Max</spotbugs.effort>
```

Comment:
Resolve conflict and provide some doc reference

Reply:
I hope this resolves the issue

Reply:
https://www.jenkins.io/doc/developer/tutorial-improve/add-more-spotbugs-checks/#increase-spotbugs-checks